components/events.tsx

```diff
@@ -35,19 +89,27 @@ const Events = () => {
 	}, [curEvent]);
 
 	const getData = () => {
-		return fetch('/api/events')
```

Comment:
Is anyone using the events API now? If not, maybe add that functionality to the events API, instead of creating a new one?